hack/builder/entrypoint.sh

```diff
@@ -3,5 +3,5 @@ set -e
 set -o pipefail
 
 source /etc/profile.d/gimme.sh
-export GOPATH="/root/go"
+export GOPATH="/go"
```

Comment:
Why is this change required? This file is not used in Dockerfile.ci, is it? Note: it IS used in another Dockerfile.

Reply:
ok, so we don't need entrypoint.sh at all then. I'll remove this change : )